examples/interfaces-and-unions/README.md

```diff
@@ -0,0 +1,129 @@
+This is a example of a service build with Pylon that uses Interfaces and Unions.
```

Comment:
There are two grammatical issues here:
- _a example_ should be _an example_
- _build_ should be _built_ to use the correct tense

Reply:
yea right. I realized to late that you already prepared a text for the example project. I will just use that.